### PR TITLE
add rerender tests, change rerender behaviour

### PR DESCRIPTION
**Pull request by kylezeeuwen**

### CODE REVIEW

File: theSrc/internal_www/data/scatterplot_four_point_brand/config.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
```
Comment (kylezeeuwen) [inline theSrc/internal_www/data/scatterplot_four_point_brand/config.json]: same as `scatterplot_three_point_brand/config.json` but added a coca-cola label

File: theSrc/scripts/LabeledScatter.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ class LabeledScatter {
                   .append('svg')
                   .attr('width', this.width)
                   .attr('height', this.height)
```
Comment (kylezeeuwen) [inline theSrc/scripts/LabeledScatter.js]: this extra class is used in the automation

File: theSrc/scripts/LabeledScatter.js [hunk 3]
```diff
@@ -54,7 +54,7 @@ class LabeledScatter {
   setConfig (data) {
     // Reset widget if previous data present but not equal in params - see VIS-278
     if (!(_.isUndefined(this.data)) && !(_.isUndefined(this.plot)) && !(this.plot.isEqual(data))) {
```
Comment (kylezeeuwen) [inline theSrc/scripts/LabeledScatter.js]: only relevant change to code. If you approve we need to get the branch tested in Displayr

Comment (poliu2s) [inline theSrc/scripts/LabeledScatter.js]: I tested the branch on reusdev and it is working as per https://numbers.atlassian.net/browse/VIS-278

Approved.

**Comment by poliu2s** [comment]
Note that there is a spec that will need to be updated

**Pull request merged**